src/Guide/Api/Utils.hs

```diff
@@ -58,6 +62,10 @@ instance (KnownSymbol help, ToSchema a) => ToSchema (a ? help) where
 instance {-# OVERLAPPING #-} (KnownSymbol help, Selector s, ToSchema c) => GToSchema (S1 s (K1 i (Maybe c ? help))) where
   gdeclareNamedSchema opts _ = fmap unnamed . withFieldSchema opts (Proxy2 :: Proxy2 s (K1 i (Maybe c ? help))) False
 
+-- | Unwraper for 'field ? help'
```

Comment:
```suggestion
-- | Unwrapper for @field '?' help@
```

Reply:
(Funky Haddock syntax; `@ @` is for monospace, `' '` is for linking to Haskell identifiers)